torch/_prims_common/__init__.py

```diff
@@ -438,32 +438,21 @@ def is_channels_last_contiguous_or_false(a: Tensor) -> bool:
     ) or is_channels_last_contiguous_or_false_3d(a)
 
 
-def is_non_overlapping_and_dense(a: Tensor) -> bool:
+def _is_non_overlapping_and_dense(sizes, strides) -> bool:
```

Comment:
call the function _is_non_overlapping_and_dense_or_false.
"""may return false instead of throwing DDE at data dependency"